docs/platform/howto/byoc/create-custom-cloud.md

```diff
@@ -587,30 +595,35 @@ AWS account. Continue working in the **Create custom cloud** wizard:
     Console](https://console.aiven.io/).
     :::
 
-3.  In your AWS account, run the template with the variables using
+1.  In your AWS account, run the template with the variables using
     Terraform.
 
     :::important
     When running `terraform plan` and `terraform apply`, make sure you
     add `-var-file=FILE_NAME.vars` as an option.
     :::
 
-4.  Find the role identifier (Role ARN) in the output script after
+1.  Find the role identifier (Role ARN) in the output script after
     running the template.
 
-5.  Enter Role ARN into the **Role ARN** field in the **Create custom
+1.  Enter Role ARN into the **Role ARN** field in the **Create custom
     cloud** wizard.
 
-6.  Select **Next** to proceed or park your cloud setup and save
+1.  Select **Next** to proceed or park your cloud setup and save
     your current configuration as a draft by selecting **Save draft**.
     You can resume creating your cloud later.
 
 #### Set up your custom cloud's availability
 
-Select in what projects you'll be able to use your new custom cloud to
-create services. You can make your cloud available for all the projects
-in your organization, selected organizational units, or specific
-projects only.
+Select in what projects you'll be able to use your new custom cloud as a hosting cloud for
+services. In the projects where you enable your custom cloud, you'll be able to create new
+services in the custom cloud and migrate your existing services to the custom cloud.
+
+You can make your cloud available as follows:
+
+- For all the projects in your organization
+- For selected organizational units
+- For specific projects only
```

Comment:
This would imply we are going into a list of steps make a cloud available, but this is not the case, so perhaps something like this:

```suggestion
You cloud can be available in:

- All the projects in your organization
- Selected organizational units
- Specific projects only
```